app/models/compliance.rb

```diff
@@ -83,7 +83,7 @@ def self.check_compliance(target, _inputs = {})
       return
     end
 
-    compliance_result = results[:actions].nil? ? true : !results[:actions].key?(:compliance_failed)
+    compliance_result = results[:action].nil? ? true : !results[:action].key?(:compliance_failed)
```

Comment:
Compare that to [this](https://github.com/ManageIQ/manageiq/commit/e5f24705b8f8b2b5ec1cd8d20b93e8d9503d3420#diff-74bb73b4bf1317b7b3582b5b962c88bcR190)